docs/odh-rhods/custom-runtime-triton.md

```diff
@@ -0,0 +1,227 @@
+# Deploying and using a Custom Serving Runtime in RHODS
+
+## Before you start
+
+This document will guide you through the broad steps necessary to deploy a custom Serving Runtime to serve a model using the triton Runtime.
+
```

Comment:
Capital T on Triton. I would also put the real name as a reference:
... using the Triton Runtime (NVIDIA Triton Inference Server).

Reply:
done in https://github.com/opendatahub-io-contrib/ai-on-openshift/pull/38/commits/0a759ddaa4597ddf7da23a67fd19537a4e4acf60